webapp/src/helpers/__tests__/meetingsHelper.ts

```diff
@@ -0,0 +1,87 @@
+import { compareMeetingNotes } from '../meetingsHelper';
+import { MeetingNote } from '../../types/api';
+
+describe('meetingsHelper', () => {
+  describe('compareMeetingNotes', () => {
+    it('should sort non-null participants first', () => {
+      expect(
+        compareMeetingNotes(
+          { agenda_owning_participant_id: 1 } as MeetingNote,
+          { agenda_owning_participant_id: null } as MeetingNote
+        )
+      ).toBeLessThan(0);
+    });
+    it('should sort null participants last', () => {
```

Comment:
What are `null` participants?  Should this description be a little more descriptive to describe the column/property that would be `null`?

Reply:
I agree, this description doesn't represent the way we think about the system. A better description would be something like, "should sort notes in participants' agendas before action items" or something like that.